web/src/overlays.js

```diff
@@ -324,7 +327,7 @@ export function open_settings() {
         name: "settings",
         $overlay: $("#settings_overlay_container"),
         on_close() {
-            browser_history.exit_overlay();
+            call_hooks(close_settings_hooks);
```

Comment:
I think actually this import cycle is better fixed by just moving this function to `settings.js` -- it's only called from there and from `admin.js` that imports `settings.js`, and none of the other things in heer have a reason to be in this file. 

Maybe `settings.open_settings_overlay()` would be a fine name for it once moved.

Reply:
Thank you, @timabbott, for suggesting this better approach. I have made the recommended changes to the code, and the pull request is now ready for review.